sdk/src/main/res/layout/activity_verloop.xml

```diff
@@ -5,7 +5,8 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     android:clipChildren="false"
-    android:orientation="vertical">
+    android:orientation="vertical"
+    android:fitsSystemWindows="true">
```

Comment:
fitsSystemWindows still exists, but its automatic padding behavior is mostly legacy now.

On Android 15 and above, you should migrate to WindowInsets handling using ViewCompat.setOnApplyWindowInsetsListener() or WindowInsetsController